Sources/ApolloWebSocket/DefaultImplementation/SSLSecurity.swift

```diff
@@ -157,9 +157,7 @@ open class SSLSecurity : SSLTrustValidator {
                 collect.append(SecCertificateCreateWithData(nil,cert as CFData)!)
             }
             SecTrustSetAnchorCertificates(trust,collect as NSArray)
-            var result: SecTrustResultType = .unspecified
-            SecTrustEvaluate(trust,&result)
-            if result == .unspecified || result == .proceed {
+            if SecTrustEvaluateWithError(trust,nil) {
```

Comment:
This logic loses the `.unspecified` (default) behaviour but I think it's better now that we take action based only on the return. We don't gather any error because it was not being logged before and I don't see any good way in this module to output it in a way that is actually useful to a production app.

_`SecTrustEvaluateWithError(:,:)` is the synchronous replacement of `SecTrustEvaluate(:,:)`. It is now a simple boolean return._